packages/notification-center/src/shared/config/themeDefaultValues.ts

```diff
@@ -0,0 +1,49 @@
+import { colors } from './colors';
+import { shadows } from './shadows';
+import { INovuTheme } from '../../store/novu-theme.context';
+
+const defaultTheme = {
+  fontFamily: 'Lato',
+};
+
+export const defaultLightTheme: INovuTheme = {
+  layoutWrap: {
+    colors: {
+      main: colors.vertical,
+      fontColor: colors.B60,
+      secondaryFontColor: colors.B80,
+    },
+  },
+  layout: { background: colors.white, boxShadow: shadows.medium },
```

Comment:
Was thinking about if we can have similar colors grouped together such as: backgroundColor, secondaryBackground and etc... 

And than do the following fallback:
1) Use specific theme: theme.header.backgroundColor
2) if 1. was not specified use general theme.backgrounColor
3) if both wasn't specified by user, use a default value of backgroundColor